.cirrus.yml

```diff
@@ -1,3 +1,29 @@
+task:
+  auto_cancellation: true
+
+  env:
+      PY_VER: "3.6"
+
+  name: Black Formatting
+  container:
+      image: python:latest
```

Comment:
this is where you'd specify which language version we're testing against so maybe you want to pin it to 3.6 as you did above?